index.js

```diff
@@ -20,16 +20,25 @@ module.exports = function(content) {
 			newContent = newContent.replace(/^\s*<\?xml [^>]*>\s*/i, "");
 		}
 
-		newContent = newContent.replace(/"/g, "'");
-		newContent = newContent.replace(/\s+/g, " ");
-		newContent = newContent.replace(/[{}\|\\\^~\[\]`"<>#%]/g, function(match) {
-			return '%'+match[0].charCodeAt(0).toString(16).toUpperCase();
-		});
+		var data;
+		if (query.base64) {
```

Comment:
I'd like to have option named "encoding" with 2 possible string values - "base64" and "none", with "none" being the default one to ensure backward compatibility.
Also please add the note about that option and its possible values to Readme, like it is done for "iesafe" option.

Reply:
Cool, makes sense. Will do.

Reply:
This is done.